include/rocksdb/cloud/cloud_env_options.h

```diff
@@ -296,14 +339,18 @@ class CloudEnv : public Env {
   // data from cloud storage.
   // If dest_bucket_name is empty, then the associated db does not write any
   // data to cloud storage.
-  static Status NewAwsEnv(Env* base_env, const std::string& src_bucket_name,
+  static Status NewAwsEnv(Env* base_env,
+			  const std::string& src_bucket_name,
                           const std::string& src_object_prefix,
                           const std::string& src_bucket_region,
                           const std::string& dest_bucket_name,
                           const std::string& dest_object_prefix,
                           const std::string& dest_bucket_region,
+                          CloudEnvOptions& env_options,
```

Comment:
Can we keep this as `const CloudEnvOptions& env_options` and make the copy inside of `NewAwsEnv`? Otherwise this is a very subtle public API breaking change. One copy won't hurt.